src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+
+export default () => {
+
+  const {article_id} = useParams()
+  const [article, setArticle] = useState({})
+
+  useEffect(()=>{
+    axios.get(`https://nc-news-ngma.onrender.com/api/articles/${article_id}`)
+    .then((response)=>{
+      setArticle(response.data.article)
```

Comment:
You should have a loading state, give the user some indication the app is working